projects/samples/environments/indoor/worlds/break_room.wbt

```diff
@@ -0,0 +1,787 @@
+    "Shelf (2, 2, 1, 0)"
+    "Shelf (1, 2, 1, 0)"
+    "Shelf (1, 1, 0, 1)"
+  ]
+  handle CabinetHandle {
+    handleColor 0.533333 0.541176 0.521569
+  }
+  primaryAppearance MattePaint {
+    baseColor 0.729412 0.741176 0.713725
+  }
+  secondaryAppearance GlossyPaint {
+    baseColor 0.643137 0 0
+  }
+}
+Cabinet {
+  translation 2.49 0 0.66
+  rotation 0 -1 0 1.5708
+  name "cabinet(3)"
+  outerThickness 0.02
+  rowsHeights [
+    0.7, 0.3, 0.3
+  ]
+  columnsWidths [
+    0.6
+  ]
+  layout [
+    "LeftSidedDoor (1, 1, 1, 1, 1.5)"
+    "Shelf (1, 3, 1, 0)"
+    "Shelf (1, 2, 1, 0)"
+    "Shelf (1, 1, 1, 1)"
+  ]
+  handle CabinetHandle {
+    handleColor 0.533333 0.541176 0.521569
+  }
+  primaryAppearance MattePaint {
+    baseColor 0.729412 0.741176 0.713725
+  }
+  secondaryAppearance GlossyPaint {
+    baseColor 0.643137 0 0
+  }
+}
+Cabinet {
+  translation 2.5 0.1 1.52
+  rotation 0 1 0 -1.5708
+  name "cabinet(4)"
+  outerThickness 0.02
+  rowsHeights [
+    0.22, 0.22, 0.22
+  ]
+  columnsWidths [
+    0.5, 0.5
+  ]
+  layout [
+    "LeftSidedDoor (2, 1, 1, 3, 1.5)"
+    "LeftSidedDoor (1, 1, 1, 3, 1.5)"
+    "Shelf (1, 1, 0, 3)"
+    "Shelf (2, 1, 0, 3)"
+  ]
+  handle CabinetHandle {
+    translation 0.26 0.02 0
+    handleLength 0.1
+    handleRadius 0.008
+    handleColor 0.427451 0.513725 0.533333
+  }
+  primaryAppearance MattePaint {
+    baseColor 0.94667 0.925551 0.852003
+  }
+  secondaryAppearance MattePaint {
+    baseColor 0.94667 0.925551 0.852003
+  }
+}
+Cabinet {
+  translation 2.5 0.1 2.31
+  rotation 0 1 0 -1.5708
+  name "cabinet(5)"
+  outerThickness 0.02
+  rowsHeights [
+    0.22, 0.22, 0.22
+  ]
+  columnsWidths [
+    0.5
+  ]
+  layout [
+    "Drawer (1, 3, 1, 1, 1)"
+    "Drawer (1, 2, 1, 1, 1)"
+    "Drawer (1, 1, 1, 1, 1)"
+    "Shelf (1, 1, 0, 3)"
+  ]
+  handle CabinetHandle {
+    translation 0.02 0.01 0
+    handleLength 0.1
+    handleRadius 0.008
+    handleColor 0.427451 0.513725 0.533333
+  }
+  primaryAppearance MattePaint {
+    baseColor 0.94667 0.925551 0.852003
+  }
+  secondaryAppearance MattePaint {
+    baseColor 0.94667 0.925551 0.852003
+  }
+}
+SolidBox {
+  translation 2.25 0.05 1.79
+  rotation 0 1 0 1.5708
+  size 1.58 0.1 0.5
+  appearance PBRAppearance {
+    baseColor 0.8 0.8 0.8
+    roughness 0.5
+    metalness 0
+    textureTransform TextureTransform {
+      scale 4 4
+    }
+  }
+}
+Solid {
+  translation 2.25 0.83 1.525
+  rotation 0 -1 0 3.14159
+  children [
+    Shape {
+      appearance PBRAppearance {
+        baseColorMap ImageTexture {
+          url [
+            "textures/rustic_veneer_wood_base_color.jpg"
+          ]
+        }
+        roughnessMap ImageTexture {
+          url [
+            "textures/rustic_veneer_wood_roughness.jpg"
+          ]
+        }
+        metalness 0
+        normalMap ImageTexture {
+          url [
+            "textures/rustic_veneer_wood_normal.jpg"
+          ]
+        }
+        normalMapFactor 0.2
```

Comment:
This deserves an appearance PROTO ;-)

Reply:
Indeed, it would be necessary to make a Wood texture that has much more impact in terms of modification. This may be the subject of a future PR.